### PR TITLE
[FLOSS H2] Added external links and clarification to websideguide

### PR DESCRIPTION
**Pull request by Nirnberger**
expanded [webside readme](https://github.com/ElektraInitiative/libelektra/tree/master/src/tools/website) to clarify.

## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

**Review by markus2330 — APPROVED** [review]
Good improvement, build server is not yet happy though.

File: doc/news/_preparation_next_release.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ _(Michael Tucek)_
 - Fix broken links. _(Robert Sowula)_
 - Emphasize that `type` is required when the HL API is used. _(Tobias Schubert @qwepoizt)_
 - Add debugging tutorial. _(Tobias Schubert @qwepoizt)_
+- Expanded the webside guid for easier understanding and linked to cmake.org. _(Philipp Nirnberger @nirnberger)_
```
Comment (muskater) [inline doc/news/_preparation_next_release.md:151]: Here might be a small typo. I believe it is "guide" not "guid".

File: src/tools/website/README.md [hunk 2]
```diff
@@ -33,7 +35,8 @@ As next step, you can build the website:
 To run the application, basically two options are available:
 
 - Use the built-in webserver of `grunt`, which can be configured in the
-  [Gruntfile.js](Gruntfile.js) and run by `grunt server` (in the installation target directory)
+  [Gruntfile.js](Gruntfile.js) and run by `grunt server` (in the installation target directory `@CMAKE_INSTALL_PREFIX@/share/elektra/tool_data/website
+`)
```
Comment (muskater) [inline src/tools/website/README.md:39]: I think that this new line is causing the error with the reformatting tool

**Review by muskater — APPROVED** [review]
This is a great addition and improvement to the website tutorial. Especially for a beginner, it is very useful to have the compiling tutorial linked.
I am pretty sure that the build server complaint because you did not reformat the new guide with _prettier_. As far as I know, in this repository every Markdown file is reformatted with it. You can check it out here:  https://prettier.io/
They have a free playground environment available, in case you do not want to install it locally.
Just open the playground and choose _Markdown_ in the left side panel under parser, then past the Markdown file into the text field.

**Review by markus2330 — CHANGES_REQUESTED** [review]
I agree with the review comments! And please also rebase.

**Pull request merged**